bgpd/bgp_advertise.c

```diff
@@ -173,7 +173,7 @@ int bgp_adj_out_lookup(struct peer *peer, struct bgp_node *rn,
 						 : (adj->attr ? 1 : 0));
```

Comment:
Should we fix this return up?

Reply:
I have this covered and lots more in https://github.com/FRRouting/frr/pull/6059

Reply:
yes this is that review, look at line 172 and 173 it's return that returns 1: 0

Reply:
ups mixed PRs :)

Reply:
ok, will fix this

Reply:
fixed